contracts/src/v0.8/dev/vrf/VRFV2PlusWrapper.sol

```diff
@@ -561,4 +562,13 @@ contract VRFV2PlusWrapper is ConfirmedOwner, TypeAndVersionInterface, VRFConsume
     require(!s_disabled, "wrapper is disabled");
     _;
   }
+
+  /***************************************************************************
+   * Section: Migration of VRFV2PlusWrapper to latest VRFV2PlusCoordinator
+   ***************************************************************************/
+
+  function migrate(address newCoordinator) external onlyOwner {
+    IVRFV2PlusMigration(address(s_vrfCoordinator)).migrate(SUBSCRIPTION_ID, newCoordinator);
+    s_vrfCoordinator = IVRFCoordinatorV2Plus(newCoordinator);
```

Comment:
I don't think this is needed. Coordinator will call `setCoordinator()` to update `s_vrfCoordinator` - https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/dev/vrf/VRFCoordinatorV2_5.sol#L683

Reply:
Ah ok, sure.